src/app/containers/FrontPageMain/index.test.jsx

```diff
@@ -1,20 +1,11 @@
 import React from 'react';
 import FrontPageMain from '.';
 import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
-import { articleDataNews, articleDataPersian } from '../Article/fixtureData';
-
-// temporary: will be removed with https://github.com/bbc/simorgh/issues/836
-const articleDataNewsNoHeadline = JSON.parse(JSON.stringify(articleDataNews));
-articleDataNewsNoHeadline.content.model.blocks.shift();
+import frontPageDataPersian from '../../../../data/prod/pidgin/frontpage';
```

Comment:
Any particular reason we have an import called `frontPageDataPersian` which is pulling data from Pidgin?

Reply:
Good spot 🤦🏻‍♂️